src/strategies/galxe-loyalty-points/index.ts

```diff
@@ -0,0 +1,49 @@
+import fetch from 'cross-fetch';
+
+export const author = 'HaynarCool';
+export const version = '0.1.0';
+
+const graphqlUrl = 'https://graphigo.stg.galaxy.eco/query';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
```

Comment:
There is no way to use the `snapshot` block in the graphql? if it is not used, results will be unpredictable

Reply:
it now supports `snapshot`